back/src/adapters/secondary/pg/migrations/1707226340043_create-agency-groups-tables.ts

```diff
@@ -0,0 +1,47 @@
+import { MigrationBuilder } from "node-pg-migrate";
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.createType("agency_group_kind", ["france-travail"]);
+  pgm.createType("agency_group_scope", [
+    "direction-territoriale",
+    "direction-régionale",
+  ]);
+
+  pgm.createTable("agency_groups", {
+    id: { type: "serial", primaryKey: true },
+    siret: { type: "char(14)", notNull: true },
+    name: { type: "text", notNull: true },
+    email: { type: "text", notNull: false },
```

Comment:
est-ce que ce ne serait pas obligatoire ?

```suggestion
    email: { type: "text", notNull: true },
```

Reply:
malheureusement non, les DT n'ont quasi jamais de mail. C'est surtout pour les DR